Standards/scs-0112-v1-volume-type-standard.md

```diff
@@ -0,0 +1,227 @@
+
+The current downside is that users without admin rights currently cannot see the recommended aspects. We want and need to change this in the OpenStack workflow through adding discoverable properties (e.g. "encrypted"="true" and "replicated"="true") that are also visible for users. This way we will be able to automatically check whether a volume type with encyrption and or replication is present. It should look like this example for encryption:
+
+```text
+openstack volume type show LUKS
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Field              | Value                                                                                                                                                        |
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| access_project_ids | None                                                                                                                                                         |
+| description        | None                                                                                                                                                         |
+| id                 | d63307fb-167a-4aa0-9066-66595ea9fb21                                                                                                                         |
+| is_public          | True                                                                                                                                                         |
+| name               | LUKS                                                                                                                                                         |
+| properties         | encrypted='true'                                                                                                                                             |
+| qos_specs_id       | None                                                                                                                                                         |
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+We want to find a way to also use the internal extra_spec for replication, when the replication is automatically done by the backend. If this is not possible, we would like to introduce another property, which has to be set by the admin, when setting the volume type. Only after that we will have the possibility to automatically check for a volume type with replication.
```

Comment:
This does not belong to this standard. It should be outsources in an own issue.

Reply:
done issue is #449